client/src/components/CartWindow/Items/Item/Item.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import store from '../../../../store';
+import removeItem from '../../../../utils/cart/removeItem';
+import updateItem from '../../../../utils/cart/updateItem';
+
+import './Item.scss';
+
+export class Item extends React.Component {
+  handleOnClickIncrement(event) {
+    if (Number(this.props.data.amount) < 99) {
+      store.dispatch(updateItem(store.getState(), this.props.data, this.props.data.color, this.props.data.size, this.props.data.amount + 1));
+    }
+  }
+
+  handleOnClickDecrement(event) {
+    if (Number(this.props.data.amount) > 1) {
+      store.dispatch(updateItem(store.getState(), this.props.data, this.props.data.color, this.props.data.size, this.props.data.amount - 1));
+    }
+  }
+
+  render() {
```

Comment:
I have no idea what data used to render this component, no one knows, except person who wrote this component